### ISSUE TITLE
Tag `borderColor` property does not work on `3.4.9`

### ISSUE THREAD

**Issue by ConnorMarble** [GeekyAnts/NativeBase#5271]
### Description

Tag `color` property does not work

### CodeSandbox/Snack link

https://snack.expo.dev/OPa8Okz-v

### Steps to reproduce

1. View the tag in the snack.expo example
2. See that the tag border color is not red

### NativeBase Version

3.4.9

### Platform

- [ ] Android
- [ ] CRA
- [X] Expo
- [X] iOS
- [ ] Next

### Additional Information

I noticed that the Tag component is undocumented. Is this an experimental component? 

The `outline` variant has an issue where the `borderColor` property does not change the color. Oddly, doing `borderBottomColor` will change the color for the bottom.

<img width="169" alt="Screen Shot 2022-08-12 at 7 45 12 PM" src="https://user-images.githubusercontent.com/68395737/184457450-e98b424d-7d08-4509-9ad5-b05dde503f74.png">

**Comment by Viraj-10** [comment]
Hi @ConnorMarble, Tag is an unreleased component please avoid using it in the production app.

**Comment by huynhtastic** [comment]
@Viraj-10 this may be related, but I am trying to change a `Button`'s `borderColor`. It works if I change the component's `borderColor` like so:

<img width="408" alt="Screen Shot 2022-08-15 at 9 41 12 PM" src="https://user-images.githubusercontent.com/9666260/184786871-5abd509a-98f2-4eb2-9105-34399cbdf68b.png">

<img width="682" alt="Screen Shot 2022-08-15 at 9 40 56 PM" src="https://user-images.githubusercontent.com/9666260/184786829-2823f23c-2a30-46ee-92c6-45e0a3c92d1d.png">

but this doesn't work if I try to do this using by defining `components` using `extendTheme`:

<img width="323" alt="Screen Shot 2022-08-15 at 9 43 38 PM" src="https://user-images.githubusercontent.com/9666260/184787225-4e9679a0-d821-4279-b120-c92a7b21023f.png">

<img width="666" alt="Screen Shot 2022-08-15 at 9 43 14 PM" src="https://user-images.githubusercontent.com/9666260/184787185-ff5b127d-3565-4540-a72b-ff7af6712cff.png">

But @ConnorMarble 's suggestion of using `borderBottomColor` works:

<img width="323" alt="Screen Shot 2022-08-15 at 9 45 00 PM" src="https://user-images.githubusercontent.com/9666260/184787369-a23b532f-7d83-4fd8-a29b-83311f15e6e9.png">

<img width="674" alt="Screen Shot 2022-08-15 at 9 45 22 PM" src="https://user-images.githubusercontent.com/9666260/184787411-dc1118e2-18a9-44f9-b431-bc3c0eacf37a.png">

I'm using v3.4.9. These images are rendered on storybook but the issue happens on my ios simulator as well.

**Comment by PragyaSingla** [comment]
@huynhtastic Even I am facing same issue after updating native-base to 3.4.13. 

Customizing components are not working. In below code, only 'solid' variant is working. It is changing font color but 'outline' and 'ghost' are not changing font color to 'black'.

Same code was working in  "native-base": "^3.3.7". 

   ```
  variants: {
        solid: () => {
          return {
            _text: {
              color: 'black',
            }
          }
        },
        outline: (): IButtonProps => {
          return {
            _text: {
              color: 'black',
            }
          }
        },
        ghost: () => {
          return {
            _text: {
              color: 'black',
            }
          }
        }
```

Thanks

**Comment by ankit-tailor** [comment]
Hi Everyone, This is the original issue #4951, You can follow this approach.

**Comment by ankit-tailor** [comment]
Hi Everyone, This issue is fixed in `3.4.14`.
